pkg/cmd/step/verify/step_verify_preinstall.go

```diff
@@ -243,6 +246,22 @@ func (o *StepVerifyPreInstallOptions) Run() error {
 		return err
 	}
 
+	if requirements.Cluster.Provider == cloud.EKS {
+		if !cluster.IsInCluster() {
+			err = amazon.EnableIRSASupportInCluster(requirements)
+			if err != nil {
+				return err
```

Comment:
The thing is, I didn't want to wrap every single error because for example these two errors already contain the context from within the function, it would be a bit too repetitive.